miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -220,10 +225,17 @@ Sample.hot = {
   },
 
   /**
+   * Returns true if a new sample of the provided SampleClass can be created without an existing parent
+   */
+  canCreateNew: function (sampleClass) {
```

Comment:
Shouldn't this now include some (not all) aliquots? sampleClass.canCreateNew probably replaces the need for this function though